journal/modernized/35-chapter-32.adoc

```diff
@@ -887,8 +889,8 @@ in days that are over and gone, to His own praise and glory.
 30th. (First day.)--The state of the weather combined
 with other circumstances to weigh down my mind,
 when looking towards attending the native meeting;
-and the secret prayer of my heart last evening was,--'`Help, O Lord,
-for your great name`'s sake, and for the sake of Him who died, the just for the unjust,
+and the secret prayer of my heart last evening was,--"`Help, O Lord,
+for your great name`"s sake, and for the sake of Him who died, the just for the unjust,
```

Comment:
this should be switched back to:

```ASCIIDOC
great name`'s sake
```

Because it's a posessive apostrophe.  Then, the double-quote should be changed to match on line 894 below.

Reply:
Haha oops!

Reply:
894 is probably where i meant to put it!